Wrappers/Python/cil/plugins/astra/operators/AstraProjector2D.py

```diff
@@ -37,30 +30,27 @@ def __init__(self, image_geometry, acquisition_geometry, device):
         :param device: The device to run on 'gpu' or 'cpu'
         :type device: string
         '''
-        super(AstraProjector2D, self).__init__(image_geometry, range_geometry=acquisition_geometry)
-        
-        self.fp = AstraForwardProjector2D(volume_geometry=image_geometry,
-                                        sinogram_geometry=acquisition_geometry,
-                                        proj_id = None,
-                                        device=device)
-        
-        self.bp = AstraBackProjector2D(volume_geometry = image_geometry,
-                                        sinogram_geometry = acquisition_geometry,
-                                        proj_id = None,
-                                        device = device)
-                           
-        
+        super(AstraProjector2D,
```

Comment:
This looked better before yapf

Reply:
I think we need more of a discussion around yapf before commiting to use it

Reply:
by all means! I will drop the autoyapf commit

Reply:
removed the autoyapf